### PR TITLE
Adding basic test-cases for #391

### PR DESCRIPTION
**Pull request by Tatyana2022**
This PR introduces test cases for #391  issue implementation: 
1. Changing_profile_in_the_settings.md
2. Highlight_vcf_with_complex_conditions.md
3. Highlight_vcf_with_simple_conditions.md
4. Highlighting_the_same_variation_with_different_conditions.md
5. Structural_variations_(BND).md
6. Bubble.md
7. DIsplaying_profiles_by_default_in_the_Settings.md
8. Added fixes for all Nikita and Masha's comments for all cases

### CODE REVIEW

**Comment by Tatyana2022** [comment]
@NShaforostov @mzueva @sidoruka  I apologize, I forgot to write right away. Could you view the test cases, please?

File: docs/test-cases/vcf-highlight/Conditions/Changing_profile_in_the_settings.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+| 2 | Go to  **Settings** -> **VCF** tab | 'VCF' tab is displayed in the Settings' window|
+| 3 | Set checkbox in **Enabled** field (if it is not set)| **'Select the profile'** field is displayed |
+| 4 | Select **Profile 1** profile in dropdown list of **'Select the profile'** field|
+| 5 | Click **Save** button | <li> Changes successfully saved <li> 'Settings' window is closed | 
+| 6 | Go to **Datasets** tab | 
+| 7 | Select dataset and vcf file from **Prerequisites** | Vcf file (**agnts3.09-28.trim.dm606.realign.vcf**) is selected in dataset (**dm6_data**) 
+| 8 | Go to **Variants** panel | |
+| 9 | Find Variation on Position **chr X:12584271**| Background of variation highlight in **'ff0000'(red)** color |
+| 10| Click on variation with Position **chr X:12584271** |Browser window is opened with vcf track|
+| 11| Look at variation on vcf track in Browser| Variation is highlighted in color  **'ff0000'(red)** along all its length in expanded and collapsed view
```
Comment (NShaforostov) [inline docs/test-cases/vcf-highlight/Conditions/Changing_profile_in_the_settings.md]: Add separate steps for switching between VCF-track view ("expanded" and "collapsed")

File: docs/test-cases/vcf-highlight/Conditions/Changing_profile_in_the_settings.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+| 12| Repeat 2-8 steps for **Profile 2** 
+| 13| Find Variation from previous profile on Position **chr X:12584271**| Conditions are changed. Background of variation **isn't highlighted** in 'ff0000' (red) color **on Variants panel** and **vcf track in Browser**|
+| 14| Find Variation on Position **chr X:12587867**| Background of variation is highlighted  in **"fff000"(yellow)** along all its length in expanded and collapsed view on **Variants panel** and **vcf track in Browser**|
```
Comment (NShaforostov) [inline docs/test-cases/vcf-highlight/Conditions/Changing_profile_in_the_settings.md]: Add separate steps for switching between VCF-track view ("expanded" and "collapsed")

File: docs/test-cases/vcf-highlight/Conditions/Changing_profile_in_the_settings.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+| 4 | Select **Profile 1** profile in dropdown list of **'Select the profile'** field|
+| 5 | Click **Save** button | <li> Changes successfully saved <li> 'Settings' window is closed | 
+| 6 | Go to **Datasets** tab | 
+| 7 | Select dataset and vcf file from **Prerequisites** | Vcf file (**agnts3.09-28.trim.dm606.realign.vcf**) is selected in dataset (**dm6_data**) 
+| 8 | Go to **Variants** panel | |
+| 9 | Find Variation on Position **chr X:12584271**| Background of variation highlight in **'ff0000'(red)** color |
+| 10| Click on variation with Position **chr X:12584271** |Browser window is opened with vcf track|
+| 11| Look at variation on vcf track in Browser| Variation is highlighted in color  **'ff0000'(red)** along all its length in expanded and collapsed view
+| 12| Repeat 2-8 steps for **Profile 2** 
+| 13| Find Variation from previous profile on Position **chr X:12584271**| Conditions are changed. Background of variation **isn't highlighted** in 'ff0000' (red) color **on Variants panel** and **vcf track in Browser**|
```
Comment (NShaforostov) [inline docs/test-cases/vcf-highlight/Conditions/Changing_profile_in_the_settings.md]: > Find Variation from previous profile

Remove "from previous profile"

File: docs/test-cases/vcf-highlight/Conditions/Highlight_vcf_with_complex_conditions.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+| 2 | Go to  **Settings** -> **VCF** tab | 'VCF' tab is displayed in the Settings' window|
+| 3 | Set checkbox in **Enabled** field (if it is not set)| **'Select the profile'** field is displayed |
+| 4 | Select **Complex Conditions** profile in dropdown list of **'Select the profile'** field|
+| 5 | Click **Save** button | <li> Changes successfully saved <li> 'Settings' window is closed | 
+| 6 | Go to **Datasets** tab | 
+| 7 | Select dataset and vcf file from **Prerequisites** | Vcf file (**agnts3.09-28.trim.dm606.realign.vcf**) is selected in dataset (**dm6_data**) 
+| 8 | Go to **Variants** panel | |
+| 9 | Find Variation on Position **chr X:12584271**| Background of variation highlight in **'0083ff'(blue)** color |
+| 10| Click on variation with Position **chr X:12584271** |Browser window is opened with vcf track|
+| 11| Look at variation on vcf track in Browser| Variation is highlighted in color  **'0083ff'(blue)** along all its length in expanded and collapsed view
```
Comment (NShaforostov) [inline docs/test-cases/vcf-highlight/Conditions/Highlight_vcf_with_complex_conditions.md]: Add separate steps for switching between VCF-track view ("expanded" and "collapsed")

File: docs/test-cases/vcf-highlight/Conditions/Highlighting_the_same_variation_with_different_conditions.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+| 2 | Go to  **Settings** -> **VCF** tab | 'VCF' tab is displayed in the Settings' window|
+| 3 | Set checkbox in **Enabled** field (if it is not set)| **'Select the profile'** field is displayed |
+| 4 | Select **Different conditions (red highlight)** profile in dropdown list of **'Select the profile'** field|
+| 5 | Click **Save** button | <li> Changes successfully saved <li> 'Settings' window is closed | 
+| 6 | Go to **Datasets** tab | 
+| 7 | Select dataset and vcf file from **Prerequisites** | Vcf file (**agnts3.09-28.trim.dm606.realign.vcf**) is selected in dataset (**dm6_data**) 
+| 8 | Go to **Variants** panel | |
+| 9 | Find Variation on Position **chr X:12585943**| Background of variation highlight in **'ff0000'(red)** color <li> Variation **isn't** highlighted in **'008000' (green)** color|
+| 10| Click on variation with Position **chr X:12585943** |Browser window is opened with vcf track|
+| 11| Look at variation on vcf track in Browser| <li> Variation is highlighted in color  **'ff0000'(red)** along all its length in expanded and collapsed view <li> Variation **isn't** highlighted in **'008000' (green)** color
```
Comment (NShaforostov) [inline docs/test-cases/vcf-highlight/Conditions/Highlighting_the_same_variation_with_different_conditions.md]: Add separate steps for switching between VCF-track view ("expanded" and "collapsed")

File: docs/test-cases/vcf-highlight/Conditions/Structural_variations_(BND).md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login to NGB as system admin user | |
+| 2 | Go to  **Settings** -> **VCF** tab | 'VCF' tab is displayed in the Settings' window|
+| 3 | Set checkbox in **Enabled** field (if it is not set)| **'Select the profile'** field is displayed |
+| 4 | Select **Structural variations** profile in dropdown list of **'Select the profile'** field|
+| 5 | Click **Save** button | <li> Changes successfully saved <li> 'Settings' window is closed | 
+| 6 | Go to **Datasets** tab | 
+| 7 | Select datasets and vcf files from **Prerequisites** | Vcf files is selected in datasets |
```
Comment (NShaforostov) [inline docs/test-cases/vcf-highlight/Conditions/Structural_variations_(BND).md]: > Vcf files is selected

Fix to "Vcf files are selected"

File: docs/test-cases/vcf-highlight/Conditions/Structural_variations_(BND).md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+| 2 | Go to  **Settings** -> **VCF** tab | 'VCF' tab is displayed in the Settings' window|
+| 3 | Set checkbox in **Enabled** field (if it is not set)| **'Select the profile'** field is displayed |
+| 4 | Select **Structural variations** profile in dropdown list of **'Select the profile'** field|
+| 5 | Click **Save** button | <li> Changes successfully saved <li> 'Settings' window is closed | 
+| 6 | Go to **Datasets** tab | 
+| 7 | Select datasets and vcf files from **Prerequisites** | Vcf files is selected in datasets | 
+| 8 | Go to **Variants** panel | |
+| 9 | Find Variation on Position **Chr5 :181050143**| Background of variation highlight in **'ff0000'(red)** color |
+| 10| Click on variation with Position **Chr5 :181050143** |Browser window is opened with vcf track|
+| 11| Look at variation on vcf track in Browser| Variation and it's label is highlighted in color  **'ff0000'(red)** along all its length in expanded and collapsed view
```
Comment (NShaforostov) [inline docs/test-cases/vcf-highlight/Conditions/Structural_variations_(BND).md]: Add separate steps for switching between VCF-track view ("expanded" and "collapsed")

File: docs/test-cases/vcf-highlight/GUI/Bubble.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+| 2 | Go to  **Settings** -> **VCF** tab | 'VCF' tab is displayed in the Settings' window|
+| 3 | Set checkbox in **Enabled** field (if it is not set)| **'Select the profile'** field is displayed |
+| 4 | Select **Bubbles** profile in dropdown list of **'Select the profile'** field|
+| 5 | Click **Save** button | <li> Changes successfully saved <li> 'Settings' window is closed | 
+| 6 | Go to **Datasets** tab | 
+| 7 | Select dataset and vcf file from **Prerequisites** | Vcf file (**agnX1.09-28.trim.dm606.realign.vcf**) is selected in dataset (**dm6_data**) 
+| 8 | Go to **Variants** panel | |
+| 9 | Find Variation on Position **chr X:12585001**| Background of variation highlight in **'ff0000'(red)** color |
+| 10| Click on variation with Position **chr X:12585001** |<li>Browser window is opened with vcf track. <li>All variations from table above are displayed on the track|
+| 11| Select 'Collapsed' view
```
Comment (NShaforostov) [inline docs/test-cases/vcf-highlight/GUI/Bubble.md]: Change to something like "At the vcf-track, select 'Collapsed' view (if it was not set previously)"

**Review by NShaforostov — CHANGES_REQUESTED** [review]
Perform minor fixes according to comments, please.

And update the Pull request description according to all introduced cases.

**Review by mzueva — APPROVED** [review]

**Pull request merged**